DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Fields/ContentPickerFieldGraphSyncer.cs

```diff
@@ -196,9 +249,14 @@ private async Task<ContentItem[]> GetLatestContentItemsFromIds(JArray contentIte
         }
 
         private object GetNodeId(ContentItem pickedContentItem, IGraphMergeContext context)
-        {
+        {  
```

Comment:
We should account for preexisting node uri prefix elsewhere too (e.g. such as in `TaxonomyFieldGraphSyncer`), but we can add support for that when it's actually required, so perhaps just add a todo for now).